index.html

```diff
@@ -0,0 +1,264 @@
+      const dropZone = document.querySelector('.dnd-container');
+      const dndDesktop = document.querySelector('.dnd-desktop');
+      const dndMobile = document.querySelector('.dnd-mobile');
+      const right = document.getElementById('right');
+
+      window.onload = function() {
+        dropZone.addEventListener('dragover', function(event) {
+          event.preventDefault();
+        })
+        dropZone.addEventListener('drop', function(event) {
+          event.preventDefault();
+          if (event.dataTransfer.items) {
+            [...event.dataTransfer.items].forEach((item, i) => {
+              if (item.kind === 'file') {
+                const file = item.getAsFile();
+                uploadFile(file);
+              }
+            });
+          } else {
+            [...event.dataTransfer.files].forEach((file, i) => {
+              uploadFile(file);
+            });
+          }
+        })
+      }
+
+      function closeNav() {
+        sidebar.style.width = '20';
+        content.classList.add('hide');
+        chevronRight.classList.remove('hide');
+        chevronLeft.classList.add('hide');
+        fullLogo.classList.add('hide');
+        miniLogo.classList.remove('hide');
+        right.style.width = '100%';
+      }
+
+      function openNav() {
+        sidebar.style.width = '400';
+        content.classList.remove('hide');
+        fullLogo.classList.remove('hide');
+        chevronRight.classList.add('hide');
+        chevronLeft.classList.remove('hide');
+        right.style.removeProperty('width');
+      }
+
+      function openFilePicker() {
+        document.getElementById('file-input').click();
+      }
+
+      function fileInputChange(e) {
+        const { files } = e;
+        if (files && files.length > 0) {
+          uploadFile(files[0]);
+        }
+      }
+
+      function uploadFile(file) {
+        const uploader = new VideoUploader({
+          file,
+          uploadToken: 'YOUR_UPLOAD_TOKEN',
+        });
+        uploader.upload();
+        dropZone.classList.add('hide');
+        uploadContainer.classList.remove('hide');
+        uploader.onProgress((e) => {
+          let uploadProgress = Math.round(
+            (e.uploadedBytes * 100) / e.totalBytes
+          );
+          if (uploadProgress >= 100) {
+            document.getElementById('upload-spinner').classList.add('hide');
+            document.getElementById('upload-done').classList.remove('hide');
+            document.getElementById('ingest-spinner').classList.add('hide');
+            document.getElementById('ingest-done').classList.remove('hide');
+          }
+        });
+        uploader.onPlayable((video) => {
+          document.getElementById('play-spinner').classList.add('hide');
+          document.getElementById('play-done').classList.remove('hide');
+          uploadContainer.classList.add('hide');
+          const player = new PlayerSdk("#PlayerContainer", {
+              id: video.videoId,
```

Comment:
we could have 
```json
   autoplay: true,
   muted: true
```
because if the player appears before the generation of the tumbnail we have a black player, maybe if it autoplays it will be a bit nicer